src/api/config/routes.rb

```diff
@@ -202,6 +202,17 @@ def self.public_or_about_path?(request)
       end
     end
 
+    resource :cloud, only: [] do
+      resources :upload, except: [:new, :show, :update, :edit, :destroy], controller: 'webui/cloud/upload_jobs' do
+        new do
+          get ':project/:package/:repository/:architecture/:filename', to: 'webui/cloud/upload_jobs#new', as: '', constraints: cons
```

Comment:
This seems to be a leftover: ``` as: '',```

Reply:
Nope, otherwise it will create wrong url helpers :/